testing/trino-product-tests/src/main/java/io/trino/tests/product/hudi/TestHudiSparkCompatibility.java

```diff
@@ -44,7 +43,7 @@ public class TestHudiSparkCompatibility
     @BeforeMethodWithContext
     public void setUp()
     {
-        bucketName = requireNonNull(System.getenv("S3_BUCKET"), "Environment variable not set: S3_BUCKET");
+        bucketName = System.getenv().getOrDefault("S3_BUCKET", "test-bucket");
```

Comment:
Please revert the unrelated change.

Reply:
I did remove this from here. Do you think it would make sense to include it in a different PR though? (Eg. `TestHiveRedirectionToHudi` has it.) Without this, the environment variable shall be set to the value which is generated by the test framework if my understanding is correct.